1 test failing with Flask==1.0

In `TestFlaskParser.test_user_validation_error_with_status_code`

```
    def __call__(self, code, *args, **kwargs):
        if not args and not kwargs and not isinstance(code, integer_types):
            raise HTTPException(response=code)
        if code not in self.mapping:
            raise LookupError('no exception for %r' % code)
>       raise self.mapping[code](*args, **kwargs)
E       werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
```

I don't understand all the underlying details, but this is related to https://github.com/pallets/flask/issues/2735, which is fixed by https://github.com/pallets/flask/pull/2739 due to be released in Flask 1.0.1. (I tested the Flask version with the fix and the tests pass.)

When removing `DEBUG` mode in `flask_app.py`, all tests pass.

```python
class TestAppConfig:
    TESTING = True
    # DEBUG = True
```

Any reason to use `DEBUG` mode when testing?

--- a/tests/apps/flask_app.py
+++ b/tests/apps/flask_app.py
@@ -9,7 +9,6 @@ from webargs.core import MARSHMALLOW_VERSION_INFO
 
 class TestAppConfig:
     TESTING = True
-    DEBUG = True
 
 hello_args = {
     'name': fields.Str(missing='World', validate=lambda n: len(n) >= 3),

